docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,44 @@
+---
+id: maintainers-guide
+title: "@open-sauced/maintainers-guide"
+sidebar_label: "@open-sauced/maintainers-guide"
+keywords:
+  - "@open-sauced/maintainers-guide"
+---
+
+Welcome to OpenSauced, Maintainers Guide for contributors who are interested in being part of the maintainer team for OpenSauced repositories. 
+
+At OpenSauced, we're empowering contributors to work in the open and supporting maintainers to make data-driven decisions for their projects.
+
+It's important to us that we maintain a healthy environment for contributors and maintainers of our projects.
+
+## How do I Join the Maintainers Team?
+- Sign up for [opensauced.pizza](https://opensauced.pizza/)
+- Join discord.
+- Prove your pizza worth!
```

Comment:
[nitpick: While "prove your pizza worth" can be humorous, it implies that a person is not worthy to join if they don't contribute enough, which can turn some people away. It also goes against your goal to make more inclusive documentation. 
```suggestion
- Join our [Discord community](https://discord.com/invite/U2peSNf23P). 
- Be involved!
```